android/src/main/java/com/dylanvann/fastimage/FastImageViewWithUrl.java

```diff
@@ -1,14 +1,38 @@
 package com.dylanvann.fastimage;
 
 import android.content.Context;
+import android.graphics.drawable.Drawable;
 
 import androidx.appcompat.widget.AppCompatImageView;
 
+import com.bumptech.glide.integration.webp.decoder.WebpDrawable;
 import com.bumptech.glide.load.model.GlideUrl;
 
+import javax.annotation.Nullable;
+
 class FastImageViewWithUrl extends AppCompatImageView {
     public GlideUrl glideUrl;
 
+    public void playAnimation() {
+        if (this.getDrawable() instanceof WebpDrawable) {
+            WebpDrawable drawable = (WebpDrawable) this.getDrawable();
+            if (!drawable.isRunning()) {
+                drawable.startFromFirstFrame();
+            }
+        }
+    }
+
+    @Override
+    public void setImageDrawable(@Nullable Drawable drawable)
+    {
+        super.setImageDrawable(drawable);
+        if (drawable instanceof WebpDrawable) {
+            WebpDrawable webpDrawable = (WebpDrawable) drawable;
+            webpDrawable.setLoopCount(1);
+            webpDrawable.stop();
```

Comment:
This is an attempt to prevent the WebP image from animating on initial load, it doesn't work though 😬